src/gunicorn_prometheus_exporter/plugin.py

```
@@ -6,19 +6,24 @@
+via Prometheus-compatible metrics.
 
+You can also subclass the Gunicorn Arbiter to capture master process events.
+Refer to `test_worker.py` and `test_metrics.py` for usage and test coverage.
 """
 
 import logging
 import os
 import time
 
 import psutil
+from gunicorn.arbiter import Arbiter
```

Ruff (0.8.2)

`gunicorn.arbiter.Arbiter` imported but unused

Remove unused import: `gunicorn.arbiter.Arbiter`

(F401)